main/src/ca/uwaterloo/flix/api/BootstrapError.scala

```diff
@@ -31,8 +31,8 @@ object BootstrapError {
     override def message(f: Formatter): String = e.message(f)
   }
 
-  case class FlixPackageError(e: PackageError) extends BootstrapError {
-    override def message(f: Formatter): String = e.message(f)
+  case class FlixPackageError(e: List[PackageError]) extends BootstrapError {
```

Comment:
Any reason to embed a list of errors here, instead of returning them to a higher level?